src/workerd/api/wpt/tsconfig.json

```diff
@@ -26,6 +26,6 @@
       "wpt:*": ["../../../wpt/*"]
     }
   },
-  "include": ["**/*.ts"],
+  "include": ["**/*.ts", "../../../wpt/*.ts"],
```

Comment:
I'm not yet able to figure out this one locally, but the error @anonrig had seems to indicate that this is needed.

Reply:
Yep, it works.